README.md

```diff
@@ -0,0 +1,13 @@
+# CiviCRM: Joinery Campaign Tools
+
+Collection of usability improvements for CiviCampaign
+
+The extension is licensed under [GPL-3.0](LICENSE.txt).
+
+## Support
+![screenshot](/images/joinery-logo.png)
+
+Joinery provides services for CiviCRM including custom extension development, training, data migrations, and more. We aim to keep this extension in good working order, and will do our best to respond appropriately to issues reported on its [github issue queue](https://github.com/twomice/com.joineryhq.campaigntools/issues). In addition, if you require urgent or highly customized improvements to this extension, we may suggest conducting a fee-based project under our standard commercial terms.  In any case, the place to start is the [github issue queue](https://github.com/twomice/com.campaigntools.easyipn/issues) -- let us hear what you need and we'll be glad to help however we can.
```

Comment:
Double-check the link URLs: the second 'github issue queue' link still has the easyipn URL.

Reply:
Resolved. Thanks!